RevenueCatUI/CustomerCenter/URLUtilities.swift

```diff
@@ -65,11 +65,15 @@ enum URLUtilities {
     }
 
     static func canOpenURL(_ url: URL) -> Bool {
+#if DEBUG
+        return true
+#else
         guard !Self.isAppExtension,
               let application = Self.sharedUIApplication else {
             return false
         }
         return application.canOpenURL(url)
+#endif
```

Comment:
curious about this part - shouldn't it still be false for app extensions even when in debug?